sql/core/src/main/scala/org/apache/spark/sql/execution/SQLExecution.scala

```diff
@@ -124,7 +136,7 @@ object SQLExecution {
             physicalPlanDescription = queryExecution.explainString(planDescriptionMode),
```

Comment:
shall we use empty string here?

Reply:
Yeah, it's okay to use an empty string instead of `parsed + e`